plugin/clusterextension/extras.go

```diff
@@ -226,12 +228,23 @@ func (e ClusterExtension) Controller() (*controller, error) {
 					if err != nil {
 						return nil, fmt.Errorf("failed to load apisersver flags: %v", err)
 					}
-					newFlag := pluginmodel.APIServerFlag{
+					newFlag := pluginmodel.CommandLineFlag{
 						Name:  f.Name,
 						Value: v,
 					}
 					apiServerFlags = append(apiServerFlags, newFlag)
 				}
+				for _, f := range p.Spec.Kubernetes.ControllerManager.Flags {
+					v, err := render.StringFrom(f.Value)
+					if err != nil {
+						return nil, fmt.Errorf("failed to load controller-manager flags: %v", err)
+					}
+					newFlag := pluginmodel.CommandLineFlag{
```

Comment:
Note: ALl the types defined in the `pluginmodel` pkg has been moved to `pkg/clusterapi` in #1473